internal/build/pipeline_state_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // NOTE(dmiller): set at runtime with:
-// go test -ldflags="-X 'github.com/windmilleng/tilt/internal/build.PipelineStateWriteGoldenMaster=1'" github.com/windmilleng/tilt/internal/build -run ^TestPipeline
+// go test -ldflags="-X 'github.com/windmilleng/tilt/internal/build.PipelineStateWriteGoldenMaster=1'" ./internal/build -run TestPipeline
```

Comment:
was this intentional?

Reply:
It was indeed! That bit of regex (^) actually didn’t work for me, and would miss all the tests. It worked after I removed it...having improved upon my own Regex abilities not one bit, though.

Reply:
FWIW, I'd guess you're using a shell that expands `^` (e.g., zsh), and it was messing up your command, and it could be fixed by wrapping it in single quotes to avoid shell expansion, i.e., `'^TestPipeline'`, but it doesn't really matter

Reply:
😯 — you're exactly correct, @landism! Thank you! TIL.